src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/PersonModal/PersonBirthday/PersonBirthday.tsx

```diff
@@ -32,16 +32,27 @@ export const PersonBirthday: React.FC<PersonBirthdayProps> = ({
     setFieldValue('birthdayYear', date?.year || null);
   };
 
+  const birthdayDate =
+    birthdayMonth && birthdayDay
+      ? DateTime.local(birthdayYear ?? 1900, birthdayMonth, birthdayDay)
+      : null;
+
+  const invalidDate =
+    birthdayMonth === 0 && birthdayDay === 0
+      ? DateTime.local(birthdayYear ?? 1900, birthdayMonth, birthdayDay)
+          .invalidExplanation !== ''
+      : false;
+
+  const backupBirthdayDate =
+    `${birthdayMonth}/${birthdayDay}/${birthdayYear}` as unknown as DateTime<boolean>;
```

Comment:
This does not follow locale rules, how to get around that?

Reply:
I've fixed this. I created a function that makes the backup date localized.